### ISSUE TITLE
ColumnsWizard

### ISSUE THREAD

**Issue by PissedFirefoxUser** [cleidigh/ThunderKdB#21]
Christopher, Please PLEEEEZ update ColumnsWizard! Its absence has been haunting me every day for months now... columns are so effed up now in TB - need to be able to set my preferred columns default, see other particular columns (like CC in the Inbox) etc. And I'llbe glad to donate a few $$ to your efforts - just tell me how! Thank you.

**Comment by cleidigh** [comment]
@PissedFirefoxUser 
I really appreciate the review using five stars . despite the fact that the extension is not updated yet.
Many people would have done the opposite.
I also appreciate the donation possibility (I have purposely not taken donations on any extensions yet, I have been waiting until I felt I had done enough beyond the original authors)

I have not abandoned this extension, I am just struggling with time.  I have also taken over import exportTools which has a huge base and lots of issues.  I am trying to get out an update for that this week.

just a couple notes:
- I am actually working full-time on Thunderbird extensions and I am also part of the extension team
- unfortunately my speed is greatly reduced on everything as I have to program by voice being disabled.  you can imagine my output is a lot lower than someone without this.
- updating columns was her was next on my list , but I am struggling with some questions on approach.
- for instance, I am curious which version you are working with.  (OS as well)
- putting aside the baseline work/time I am fairly confident that I can get it to work for 68
- 74+ introduces many limitations so I really have to investigate whether there are features that require new APIs to manipulate the columns.  this is important because if I have to do something completely different I would like to know if time.

Bottom line is this, I just have to get there.  I may contemplate a rough version (I am not sure what that would look like , but sometimes there are shortcuts one can take)

Final note:  I have already worked on several of the base issues so I actually have some work towards 68 already.
Thanks also for posting here the repository - really the only way to have a conversation.
I will post here when I have something to look at, even an early prototype might help you and others

Cheers
@cleidigh

**Comment by PissedFirefoxUser** [comment]
Chris, the 5-star review was always deserved in my opinion, from the day I became CW's first user back in 2013. I'm just an ordinary TB user, running version 68.7.0 (32-bit) on Windows 10 (unfortunately) Pro for Workstations, 64-bit, version 1903, OS Build 18362.720. And I'm serious about donating to your time, I'd like to help. I'm old fashioned - PayPal, or a check by snail mail. I'm completely blown away by the fact that you have to do everything by voice - that's mind boggling. Most people really don't appreciate how they have it until something goes awry.
